email/1.3/modules/ROOT/edits/email-list.adoc

```diff
@@ -0,0 +1,210 @@
+= List Emails with Email Connector Examples - Mule 4
+:page-aliases: connectors::email/email-list.adoc
+
+Anypoint for Email (Email Connector) provides the *List - IMAP* and *List - POP3* operations to list emails from any part of the flow. The following examples illustrate how to configure these operations for the connector:
+
+* <<send-attachments,List Emails from an IMAPS Server>> +
+Configure the *List - IMAP* operation to list all the emails in the configured IMAP mailbox folder from an IMAPS server that has a TLS connection. Additionally, set a *For Each* scope component and two *Set Variable* components to iterate over the emails and save the email body and attachment content in variables.
+
+* <<send-attachments,List Emails from a POP3 Server>> +
+Configure the *List - POP3* operation to list all the emails in the configured POP3 mailbox folder that matched the criteria specified in the *Match with* parameter.
+
+These example uses variables for some field values. You can either: +
+
+* Replace the variables with their values in the code.
+* Provide the values for each variable in a properties file and then refer to that file from the connector configuration. +
+If you don’t know how to use a properties file, see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders].
+
+== List Emails from an IMAPS Server
+
+The following example illustrates how to list all the emails in the configured IMAP mailbox folder from an IMAPS server that has a TLS connection. A *Scheduler* component initiates the flow, and then the *List - IMAP* operation lists the emails. Subsequently, a *For Each* scope component and two *Set Variable* components iterate over the emails, and save the body and attachments of the email content payload in variables:
```

Comment:
ditto re. bold > plain init cap for names of things, not UI labels, which are quoted correctly in the steps (when the user is actually in the UI).